### ISSUE TITLE
Module not found: Can't resolve 'react-context-stateful' in '.../src/...'

### ISSUE THREAD

**Issue by icorbrey** [icorbrey/react-context-stateful#1]
Package is listed in both `package.json` and `package-lock.json`, and the module shows up in `node_modules`. VS Code throws no errors, but when the project is started, the following is output:

```
Failed to compile.

./src/...
Module not found: Can't resolve 'react-context-stateful' in '.../src/...'
```

**Comment by icorbrey** [comment]
Fixed by deleting project folder, recloning, and running `npm install`.

**Comment by icorbrey** [comment]
Haha just kidding, I'm stupid. Issue still occurs.

**Comment by icorbrey** [comment]
Resolved in 1.0.1.
